cfssl/models/policy_use.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License MIT (https://opensource.org/licenses/MIT).
+
+
+class PolicyUse(object):
+    """ It provides a Certificate Use policy compatible with CFSSL """
+
+    def __init__(self, name, code):
```

Comment:
I went this direction because everything is explicitly required. The whole point of these classes is to provide a data structure for usage, getting lazy when creating that structure just moves us back into what we had.